packages/remix/src/generators/application/lib/ignore-vite-temp-files.ts

```diff
@@ -0,0 +1,75 @@
+import { ensurePackage, readJson, stripIndents, type Tree } from '@nx/devkit';
+import { getPackageVersion } from '../../../utils/versions';
+
+export async function ignoreViteTempFiles(
```

Comment:
I decided to duplicate this helper for now. It should not change much in the future, and I'd like to avoid exposing an internal util from the `@nx/vite` package, which currently doesn't expose any utils. The `@nx/remix:application` generator has a custom vite setup, so it doesn't use the `@nx/vite:init` generator.